cmd/juno/dbcmd.go

```diff
@@ -97,6 +99,12 @@ func dbInfo(cmd *cobra.Command, args []string) error {
 		return fmt.Errorf("failed to get the state update: %v", err)
 	}
 
+	schemaMeta, err := migration.SchemaMetadata(database)
+	if err != nil {
+		return fmt.Errorf("failed to get schema metadata: %v", err)
+	}
```

Comment:
How much attention should we pay to this warning?

Reply:
it's just an error that most likely indicate that there are some